src/Mustaches.ts

```diff
@@ -16,8 +23,7 @@ class Mustaches {
     destination,
     name,
     data,
-    formatters = {},
-    export: exportType
+    formatters = {}
   }: InterpolationOptions): Promise<ID> {
     // If no destination given, use same folder as source
     destination = destination || (await this.getParent(source))
```

Comment:
Is this still needed?

Reply:
Yes, you may want to put the export in another folder

Reply:
Sure, but you don't need the destination in `interpolate` anymore

Reply:
We still need it for the interpolated Google Doc file: Template file => Interpolated file => Exported file. The interpolated and exported files need to be put somewhere

Reply:
Oh yes sorry, I forgot this "intermediate" file, my bad